strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RebuildMavenMetadataCronJobTestIT.java

```diff
@@ -64,38 +61,28 @@ public void init(TestInfo testInfo)
         super.init(testInfo);
     }
 
-    private Set<MutableRepository> getRepositories(TestInfo testInfo)
-    {
-        Set<MutableRepository> repositories = new LinkedHashSet<>();
-        repositories.add(createRepositoryMock(STORAGE0,
-                                              getRepositoryName("rmmcjtit-snapshots", testInfo),
-                                              Maven2LayoutProvider.ALIAS));
-        repositories.add(createRepositoryMock(STORAGE0,
-                                              getRepositoryName("trmir-snapshots", testInfo),
-                                              Maven2LayoutProvider.ALIAS));
-
-        return repositories;
-    }
-
-    @AfterEach
-    public void removeRepositories(TestInfo testInfo)
-            throws Exception
-    {
-        removeRepositories(getRepositories(testInfo));
-    }
-
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class, ArtifactManagementTestExecutionListener.class })
     @Test
-    public void testRebuildArtifactsMetadata(TestInfo testInfo)
+    public void testRebuildArtifactsMetadata(@MavenRepository(storageId = STORAGE0,
+                                                              repositoryId = RMMCJTIT_SNAPSHOTS)
```

Comment:
`policy = RepositoryPolicyEnum.SNAPSHOT`

Reply:
this policy applies only for this particular test case?